.github/workflows/flake8.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         run: |
           if [ -n "$CHANGED_FILES" ]; then
             echo "Running Flake8 on changed files: $CHANGED_FILES"
-            flake8 --inline-quotes='"' --max-line-length=88 --ignore=E231,E501,E225,E226,E261,E401,E306,E293,W293,E741,W291,E265,E302,E305,E203,W503,Q002 $CHANGED_FILES
+            flake8 --inline-quotes='"' --max-line-length=79 --ignore=E203,E225,E226,E231,E261,E265,E293,E302,E305,E306,E401,E731,E741,Q002,W291,W293,W503 $CHANGED_FILES
```

Comment:
please leave this alone -- I realized this got committed with the other stuff rather than pushed to master, but it is a needed update.